src/App/BrownBag/View.elm

```diff
@@ -4,18 +4,18 @@ import Html exposing (..)
 import Html.Attributes exposing (..)
 import Common.Utils exposing (brandUrl)
 import App.BrownBag.Messages exposing (Msg(..))
-import App.BrownBag.Models exposing (Presenter, Status(..))
+import App.BrownBag.Models exposing (BrownBag, Status(..))
```

Comment:
Totally agree with you @waiyaki. I believe we chose the `presenter` identifier because our user model was not harmonized for the other parts of the application (hangouts and santa).